python/hsfs/util.py

```diff
@@ -139,6 +140,26 @@ def get_hudi_datestr_from_timestamp(timestamp):
     return datetime.fromtimestamp(timestamp / 1000).strftime("%Y%m%d%H%M%S%f")[:-3]
 
 
+def convert_event_time_to_timestamp(event_time, time_zone=timezone.utc):
+    if event_time is None:
+        return None
+    if isinstance(event_time, str):
+        if event_time:
+            return get_timestamp_from_date_string(event_time, time_zone)
+    if isinstance(event_time, pd._libs.tslibs.timestamps.Timestamp):
```

Comment:
Can we add support for `datetime.date` and `datetime.datetime` here as well?
For `datetime.date` we can assume the timezone given in the `time_zone `parameter given to `convert_event_time_to_timestamp.`
For date.datetime we can do the same when no tzinfo is set. If the datetime has tzinfo set, we convert it sensitive of that timezone.